yt/visualization/plot_modifications.py

```diff
@@ -914,13 +950,23 @@ def __init__(
         self.factor = _validate_factor_tuple(factor)
         self.clim = clim
         self.take_log = take_log
-        if plot_args is None:
-            plot_args = def_plot_args
-        self.plot_args = plot_args
+
+        if plot_args is not None:
+            issue_deprecation_warning(
+                "`plot_args` is deprecated. "
+                "You can now pass arbitrary keyword arguments instead of a dictionary.",
+                since="4.3",
+                stacklevel=5,
+            )
+        self.plot_args = {
+            "colors": "black",
+            "linestyles": "solid",
+            **(plot_args or {}),
+            **kwargs,
+        }
+
         self.label = label
-        if text_args is None:
-            text_args = def_text_args
-        self.text_args = text_args
+        self.text_args = {"colors": "white", **(text_args or {})}
```

Comment:
this fixes an unrelated bug where def_text_args is completely ignored if any text_arg is supplied.